### PR TITLE
Simplify the way the command are built

### LINKED ISSUE #10: Rework execute_action_group
See https://github.com/iMicknl/python-tahoma-api/blob/master/tahoma_api/client.py#L104

I suggest renaming this method to *send_commands*. It will be more user friendly.

- [ ] This method must take also a device url as parameter. It's not the case. 
- [ ] A list of Command is taken as a parameter, must not convert as a json when set in the payload.

```python
{"label":"Volet du bureau - Régler à 28% - TaHoma PC","actions":[{"deviceURL":"io://xxxxxx:5978000","commands":[{"name":"setClosure","parameters":[28]}]}]}
```

### PR DESCRIPTION
**Pull request by tetienne**
Fix #10

Example usage:

```Python
device_url = 'io://xxxxx/
await client.send_command(device_url,  Command("setPosition", [23]))
await client.send_command(device_url, "close")
```

### CODE REVIEW

File: tahoma_api/client.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ async def login(self) -> bool:
         Caller must provide one of [userId+userPassword, userId+ssoToken, accessToken, jwt]
         """
         payload = {"userId": self.username, "userPassword": self.password}
-        response = await self.__post("login", payload)
+        response = await self.__post("login", data=payload)
```
Comment (tetienne) [inline tahoma_api/client.py:47]: login data are sent as form.

File: tahoma_api/models.py [hunk 2]
```diff
@@ -105,21 +105,18 @@ def __init__(self, name: str, value: str, type: str, **_: Any):
         self.type = type
 
 
-class Command:
+class Command(dict):  # type: ignore
```
Comment (tetienne) [inline tahoma_api/models.py:108]: Ease the serialization of the object. See send_commands.

File: tahoma_api/client.py [hunk 3]
```diff
@@ -141,16 +117,41 @@ async def get_current_executions(self) -> List[Any]:
...
+        commands: List[Command],
+        label: Optional[str] = "python-tahoma-api",
+    ) -> str:
+        """ Send several commands in one call """
+        payload = {
+            "label": label,
+            "actions": [{"deviceURL": device_url, "commands": commands}],
+        }
+        response = await self.__post("exec/apply", payload)
+        return response["execId"]
```
Comment (tetienne) [inline tahoma_api/client.py:141]: On purpose I've not reintroduce the supported_modes. If you know there are used, I can add it.

File: tahoma_api/client.py [hunk 4]
```diff
@@ -141,16 +117,41 @@ async def get_current_executions(self) -> List[Any]:
...
             return await response.json()
 
-    async def __post(self, endpoint: str, payload: Optional[JSON] = None,) -> Any:
+    async def __post(
+        self, endpoint: str, payload: Optional[JSON] = None, data: Optional[JSON] = None
+    ) -> Any:
         """ Make a POST request to the TaHoma API """
         async with self.session.post(
-            f"{self.api_url}{endpoint}", data=payload
+            f"{self.api_url}{endpoint}", data=data, json=payload
```
Comment (tetienne) [inline tahoma_api/client.py:154]: json argument force the content-type to json and serialize the object for us.

File: tahoma_api/models.py [hunk 5]
```diff
@@ -105,21 +105,18 @@ def __init__(self, name: str, value: str, type: str, **_: Any):
...
-        "type",
         "name",
         "parameters",
-        "sensitive_parameters_indexes",
-        "authentication",
-        "delay",
     )
 
-    def __init__(self, name: str, parameters: str, **_: Any):
+    def __init__(self, name: str, parameters: Optional[str] = None, **_: Any):
```
Comment (tetienne) [inline tahoma_api/models.py:116]: close command for instance has not parameter.

File: tahoma_api/client.py [hunk 6]
```diff
@@ -141,16 +117,43 @@ async def get_current_executions(self) -> List[Any]:
 
         return response
 
+    async def send_command(
```
Comment (tetienne) [inline tahoma_api/client.py:120]: @iMicknl Should I rename it execute_command?

Comment (tetienne) [reply tahoma_api/client.py]: Done.

File: tahoma_api/client.py (hunk 3 above)
Comment (iMicknl) [reply tahoma_api/client.py:143]: Not at the moment, but in the end, the highPriority and internal could be useful.

Comment (tetienne) [reply tahoma_api/client.py:143]: So, do you want I add them again?

**Pull request merged**